docs/guides/server/caching.adoc

```diff
@@ -123,7 +123,6 @@ Follow these steps to enable this setup:
 1. Since the cache is the only source of truth for user and client sessions, configure caches to not limit the number of entries and to replicate each entry to at least two nodes. To do so, edit {project_name}'s cache config file (`conf/cache-ispn.xml`) for caches `sessions` and `clientSessions` with the following update:
   * Remove the `+<memory max-count="..."/>+`
   * Change `owners` attribute of the `distributed-cache` tag to 2 or more
-
```

Comment:
I am not sure this is correct. 

What I was trying to achieve with the empty line is to have the line:
`An example of the resulting configuration for the `sessions` cache would look as follows.` indented 
 the same way as  the parent item `Since the cache is the o.....` instead of `Change `owners` attribute of the `distributed-cache` tag to 2 or more` subitem.

Reply:
In my IDE it looks like this with this change:
![image](https://github.com/user-attachments/assets/209bac8a-4831-4314-8aaa-83bfa603db76)